waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -476,6 +479,7 @@ public void create_table_with_constraints(Table tbl, List<SQLPrimaryKey> primary
   @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
   public void drop_table(String dbname, String name, boolean deleteData)
       throws NoSuchObjectException, MetaException, TException {
+    name = MetaStoreUtils.parseDbName(name, conf)[1];
```

Comment:
I think added testcases will help getting understanding what you're trying to fix.

Reply:
I am truly sorry, this should be dbname instead of name. I will modify the file content as soon as possible

Reply:
I meant in general I'm not sure what sure what this code does without reading it:  `MetaStoreUtils.parseDbName`
It will help if you write unit test for the changes you made so it's easier to understand why you made them. Thanks!

Reply:
I will add corresponding unit tests for these modifications as soon as possible to improve their readability.

Reply:
I'm sorry, for some reason, I confused the latest code generated tarball with the previous tarball, which led me to believe that the latest commit did not solve this problem.
The above issue has been resolved in a345e74e0a5120bbff03ac0fd90cab4d4ee74809. For the case where **@hive#default** carries catalog information, it uses `GrammarUtils.removeCatName(databaseName)` method to remove the catalog information. 
Thank you for your enthusiastic reply and suggestions. In the future, I will be more cautious with every PR I receive.